packages/build/src/packaging/download-crypt-library.ts

```diff
@@ -38,6 +38,9 @@ export async function downloadCryptLibrary(
   if (/ppc64|s390x/.test(opts.arch || process.arch)) {
     versionSpec = 'stable';
   }
+  if ((opts.platform || process.platform) === 'darwin') {
+    versionSpec = '8.0.5'; // TBD(SERVER-101020): Figure out at what point we use a later version.
```

Comment:
Just for posterity, this TBD is for now just an FYI message to the relevant product management in https://mongodb.slack.com/archives/C072HR5KQA1/p1743004802006709?thread_ts=1741961998.278709&cid=C072HR5KQA1